include/stddef.h

```diff
@@ -14,3 +14,4 @@ typedef struct {
 
 #define NULL ((void*)0)
 #define offsetof(T, member) __builtin_offsetof(T, member)
+#define bitoffsetof(T, member) __builtin_bitoffsetof(T, member)
```

Comment:
We probably shouldn't define this, because it's not a reserved identifier and not part of the C standard. User code might use `bitoffsetof` and it shouldn't break due to including `stddef.h`.

Reply:
nice catch.